AssetManagementSystem/appsettings.json

```diff
@@ -5,5 +5,8 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "dbstring": "Server=DASUN-THEEKSHAN;Database=AssetManegementOfficial;Trusted_Connection=True;TrustserverCertificate=True"
```

Comment:
for now, let's make this string a empty when we commit.. i will find a better solution later

Reply:
I made the server name empty.
is it okay or do I need to empty the string fully?

Reply:
okay lets keep this as it is.